src/main/webapp/index.html

```diff
@@ -4,27 +4,45 @@
     <meta charset="UTF-8">
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/scripts/mapscript.js"></script>
+    <script type='text/javascript' src='/scripts/config.js'></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDME967qxzDl6dYJTT_d25n6yPHM1LOehw&callback=initMap&libraries=&v=weekly"
+      defer
+    ></script>
     <link rel="stylesheet" href="/styles/fonts.css">
     <link rel="stylesheet" href="/styles/common.css">
+    <link rel="stylesheet" href="/styles/recmap.css">
   </head>
-  <body>
+  <body onload="initMap()">
     <div class="header">
-      <h1>StudentRecs</h1>
+      <h1>Student/Recs</h1>
       <h3>a recommendation system for students, by students!</h3>
-
+      <!-- Navigation Bar -->
       <nav>
         <ul>
           <li><a class="active" href="#recmap">Recommendation Map</a></li>
           <li><a href="#user">User Directory</a></li>
         </ul>
       </nav>
+    </div>    
+    <h3>Recommendation Map</h3>
+    <div class="map-category-section">
+      <button class="map-category-button">Restaurants</button>
+      <button class="map-category-button">Places to Visit</button>
+      <button class="map-category-button">Bars and Clubs</button>
+      <button class="map-category-button">Study Places</button>
     </div>
-    <div class="content">
-      <h1>TODO: add content</h1>
+    <!-- The Recommendation Map -->
+    <div id="map"></div>
+    
+    <div class="map-info">
```

Comment:
Is this class name used anywhere else?

Reply:
This is going to be reused but good point, if it isn't used anywhere else, I'll update the code :)